shared/build.gradle.kts

```diff
@@ -34,29 +36,23 @@ kotlin {
         val iosX64Main by getting
         val iosArm64Main by getting
         val iosSimulatorArm64Main by getting
```

Comment:
remove it

Reply:
Do you mean iosX64Main -> iosSimulatorArm64Main can be removed?

Reply:
yes